tests/main/views/test_frameworks.py

```diff
@@ -244,15 +250,27 @@ def setup(self):
             'clarificationQuestionsOpen': False,
             'lots': ['saas', 'paas', 'iaas', 'scs'],
             'applicationCloseDate': '2023-04-11T16:00:00.000000Z',
+            'applicationsCloseAtUTC': '2023-04-11T16:00:00.000000Z',
+            'intentionToAwardAtUTC': '2023-04-25T00:00:00.000000Z',
+            'clarificationsCloseAtUTC': '2023-03-30T17:00:00.000000Z',
+            'clarificationsPublishAtUTC': '2023-04-04T17:00:00.000000Z',
+            'frameworkLiveAtUTC': '2023-05-01T00:00:00.000000Z',
+            'frameworkExpiresAtUTC': '2000-06-06T00:00:00.000000Z',
```

Comment:
Slightly irrelevant, but did you mean to put `'2023-06-06T00:00:00.000000Z'`

Reply:
Ha. I knew you were going to spot this.

Reply:
Updated.